collect_app/src/test/java/org/odk/collect/android/google/GoogleAccountsManagerTest.java

```diff
@@ -150,7 +150,7 @@ public void displayAccountPickerDialogInAutoChooseWhenNoAccountIsNotAvailableTes
     }
 
     @Test
-    public void permissionsNotAvailableTest() {
+    public void shouldRequestForPermissionIfPermissionNotGivenTest() {
```

Comment:
Perfect! Technically we don't need 'Test' at the end anymore (since JUnit4) but there's no need to change that now.